content/en/docs/reference/glossary/probe.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Probe
+id: probe
+date: 2023-03-21
+full_link: /docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+
+short_description: >
+  A check performed periodically by the kubelet on a container in a Pod.
+
+tags:
+- tool
+---
+A check that the kubelet periodically performs against a container that is 
+running in a pod to define container state and health and informing container lifecycle.
```

Comment:
👋 @KhushPatibandha .
Great start! Could you break up the main sentence? The last phrase beginning with "to define container state and ..." is not clear.